services/ingest/app/routers/api.py

```
@@ -644,3 +651,96 @@ async def alerts_inbound(
 
     await session.commit()
     return {"status": "ok", "resolved": resolved_count}
+
+
+@router.post("/deployments/notify", response_model=DeploymentNotifyResponse)
+async def notify_deployment(
+    body: DeploymentNotifyRequest,
+    session: AsyncSession = Depends(get_session),
```

Ruff (0.16.3)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)